<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileBrowser</class>
 <widget class="QWidget" name="FileBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="currentPathLabel">
        <property name="text">
         <string>Path: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathLineEdit">
        <property name="toolTip">
         <string>Pree Enter to go to path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sequenceCheckBox">
        <property name="toolTip">
         <string>Group sequences of files into one entry</string>
        </property>
        <property name="text">
         <string>Sequence as one entry</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,5">
     <item>
      <widget class="QTreeView" name="dirTreeView">
       <property name="lineWidth">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="fileSequenceTableWidget">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::NoPen</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dirTreeView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>FileBrowser</receiver>
   <slot>updateFileViewer(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pathLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>FileBrowser</receiver>
   <slot>editPathPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>888</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sequenceCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>FileBrowser</receiver>
   <slot>toggleShowSequences(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>760</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>887</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateFileViewer(QModelIndex)</slot>
  <slot>editPathPressed()</slot>
  <slot>toggleShowSequences(bool)</slot>
 </slots>
</ui>
